### PR TITLE
Add pool.yaml, pool-ec.yaml and storageclass-ec.yaml

### PR DESCRIPTION
**Pull request by ebondare**

### CODE REVIEW

File: templates/CSI/cephfs/pool.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+#################################################################################################################
+# Create a Ceph pool with settings for replication in production environments. A minimum of 3 OSDs on
+# different hosts are required in this example.
+#  kubectl create -f pool.yaml
+#
+#  Origin: https://github.com/rook/rook/blob/master/cluster/examples/kubernetes/ceph/pool.yaml
+#################################################################################################################
+
+apiVersion: ceph.rook.io/v1
+kind: CephBlockPool
```
Comment (ebenahar) [inline templates/CSI/cephfs/pool.yaml]: I think this yaml should be under CSI/rbd

File: templates/CSI/cephfs/storageclass-ec.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+# Create a storage class with a data pool that uses erasure coding for a production environment.
+# A metadata pool is created with replication enabled. A minimum of 3 nodes with OSDs are required in this
+# example since the default failureDomain is host.
+#  kubectl create -f storageclass-ec.yaml
+#
+#  Origin: https://github.com/rook/rook/blob/master/cluster/examples/kubernetes/ceph/storageclass-ec.yaml
+#################################################################################################################
+
+apiVersion: ceph.rook.io/v1
+kind: CephBlockPool
```
Comment (ebenahar) [inline templates/CSI/cephfs/storageclass-ec.yaml]: same

File: templates/CSI/cephfs/pool.yaml (hunk 1 above)
Comment (vasukulkarni) [reply templates/CSI/cephfs/pool.yaml]: yep should be in CSI/ folders. thanks

Comment (ebondare) [reply templates/CSI/cephfs/pool.yaml]: Done.

File: templates/CSI/cephfs/storageclass-ec.yaml (hunk 2 above)
Comment (ebondare) [reply templates/CSI/cephfs/storageclass-ec.yaml]: Done

**Pull request merged**